### ISSUE TITLE
Sync

### ISSUE THREAD

**Issue by TheGer** [6samurai/Staterra#1]
Guys, when you have time, pls sync both branches so I can see your latest commits

Thanks

G

**Comment by FaeLD** [comment]
I should have synced my branch with a new commit too. Let me know if it didn't work

**Comment by TheGer** [comment]
so I added this

//change the position by local scale
		eventData.position = new Vector3 (eventData.position.x, eventData.position.y) / GameObject.FindObjectOfType<Canvas> ().GetComponent<RectTransform> ().localScale.x;

to DragMe just before

SetDraggedPosition(eventData);

basically I divided the position being thrown by the mouse by the scaling factor of the local scaler.  

It worked fine when I built it

**Comment by FaeLD** [comment]
Gave me this error

[image: Inline images 1]

**Comment by TheGer** [comment]
the code came out wrong, but I sent you an email with a link to the working example

```
    //change the position by local scale
    eventData.position = new Vector3 (eventData.position.x, eventData.position.y) /   GameObject.FindObjectOfType<Canvas> ().GetComponent<RectTransform> ().localScale.x;

    SetDraggedPosition(eventData);

```

**Comment by FaeLD** [comment]
The code isn't giving me an error any more, but it still has the drag
problem in non-native res builds.

Same thing with the project .rar you sent me.

(tried both 800_600 and 1024_768 on both)

**Comment by TheGer** [comment]
it's weird, on mine works fine.  Can you check that your canvas scaler settings are the same as mine? \

UI scale mode: scale with screen size
Reference resolution : 1920/1080
Screen match mode : expand
Reference pixels per unit: 100

**Comment by FaeLD** [comment]
Yepp, I was just going to email to tell you that I tried your canvas
settings on my project and the problem is still there.

Also, Chris should have fixed your rights on github.

**Comment by TheGer** [comment]
Yep sorted.  There has to be something different as I'm trying it with the same resolutions on my desktop pc which is at 1920x1080 but am running the executable in every different resolution with no issue.   Didn't try it without the change in code though.  

Gonna try my code on my laptop to see if it's the same.  In case next week will bring my laptop and we can figure out what the issue is over there.

**Comment by FaeLD** [comment]
Sounds great, thanks. I'm going to upload a new commit only because I
normalised the code (basically just changed a variable name to get rid of
the annoying warning at the beginning of every run). And just to make sure
that all our settings are the same.

Small side note: The project files I'm committing will have 768_432 as
reference resolution. I tried it with your settings as well (1920_1080) but
it still didn't work. I'm keeping it in this reference on mine solely for
aesthetic reasons.

Thanks anyway though, we'll see you Wednesday!

**Comment by TheGer** [comment]
ok cya :)
